docs/pages/get-started/installation.mdx

```diff
@@ -18,6 +18,7 @@ To use Expo CLI, you need to have the following tools installed on your develope
 
 - [Node.js LTS release](https://nodejs.org/en/) - Only Node.js LTS releases (even-numbered) are recommended.
   As Node.js [officially states](https://nodejs.org/en/about/releases/), "Production applications should only use Active LTS or Maintenance LTS releases".
+  - Consider installing Node.js using [nvm](https://github.com/nvm-sh/nvm) or [nvm-windows](https://learn.microsoft.com/en-us/windows/dev-environment/javascript/nodejs-on-windows#install-nvm-windows-nodejs-and-npm) to easily install and switch between different versions
```

Comment:
```suggestion
  - You can install Node.js using a version management tool such as [nvm](https://github.com/nvm-sh/nvm) or [nvm-windows](https://learn.microsoft.com/en-us/windows/dev-environment/javascript/nodejs-on-windows#install-nvm-windows-nodejs-and-npm) that allows switching between different Node.js versions.
```

Let's improve the context of why one can use a tool like NVM.

Reply:
If we would like to stick to recommending any tool, it probably should be `volta`, since we use it in few our repositories and we know it works with. Additionally, it would be nice to recommend a tool which is fully cross-platform, rather then mentioning two options based on user OS (Volta also check this box - https://docs.volta.sh/guide/getting-started).

Other option could be just to list few options without a links to do not clobber the sentence:
```suggestion
  - You can install Node.js using a version management tool such as `asdf`, `n`, `nvm`  or `volta` that allows switching between different Node.js versions.
```

Reply:
I like this idea of general purpose recommendations in this case. I'm also happy with recommending volta as one route. 

@kbrandwijk, wdyt?